src/lib.rs

```diff
@@ -292,6 +293,30 @@ impl From<JsonFrontendError> for Error {
 ///
 /// [`BpfProgram`]: type.BpfProgram.html
 pub fn apply_filter(bpf_filter: BpfProgramRef) -> Result<()> {
+    apply_filter_with_flags(bpf_filter, SeccompFlagset::new())
+}
+
+/// Apply a BPF filter to the all threads in the process.
```

Comment:
Please mention that the user should search the man page containing the `SECCOMP_FILTER_FLAG_TSYNC ` flag, as there are some important considerations there